### PR TITLE
Added a command to set the ramp to the top at the beginning of the match

### PR DESCRIPTION
**Pull request by neo-Aditya**
Tested and works

### CODE REVIEW

File: src/main/java/frc/robot/subsystems/Ramp.java [hunk 1]
```diff
@@ -74,6 +74,14 @@ public double getRampPos() {
         return encoder.getPosition();
     }
 
+    public boolean getForwardSwitch() {
```
Comment (noahheller) [inline src/main/java/frc/robot/subsystems/Ramp.java]: Method name is unclear (what does true/false mean, ie pressed or not). I would recommend adding Javadocs and/or renaming method

File: src/main/java/frc/robot/commands/ResetRamp.java [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  // Called when the command is initially scheduled.
+  @Override
+  public void initialize() {
+    startTime = Timer.getFPGATimestamp();
+  }
+
+  // Called every time the scheduler runs while the command is scheduled.
+  @Override
+  public void execute() {
+    ramp.changeRampPos(99);
```
Comment (bengold22) [inline src/main/java/frc/robot/commands/ResetRamp.java]: This seems like a strange way to do this. First, how do you know which way is "forward"? Should this 99 be a constant? Why 99? Why not 9999?

File: src/main/java/frc/robot/commands/ResetRamp.java
Comment (bengold22) [inline src/main/java/frc/robot/commands/ResetRamp.java:1]: Leave a TODO in this code about how it actually needs to be tested, so we'll know what to do when we get the actual robot.

File: src/main/java/frc/robot/Robot.java [hunk 3]
```diff
@@ -62,6 +63,7 @@ public void autonomousInit() {
     @Override
     public void autonomousPeriodic() {
         loopTime = Timer.getFPGATimestamp();
+        new ResetRamp(robotContainer.getRamp());
```
Comment (bengold22) [inline src/main/java/frc/robot/Robot.java]: Putting this in autonomousPeriodic does not seem correct. You would be rescheduling this over and over, 50 times a second.

File: src/main/java/frc/robot/commands/ResetRamp.java
Comment (neo-Aditya) [reply src/main/java/frc/robot/commands/ResetRamp.java:1]: Just added it.

File: src/main/java/frc/robot/Robot.java (hunk 3 above)
Comment (neo-Aditya) [reply src/main/java/frc/robot/Robot.java]: Moved it to AutonomousInit

File: src/main/java/frc/robot/commands/ResetRamp.java (hunk 2 above)
Comment (neo-Aditya) [reply src/main/java/frc/robot/commands/ResetRamp.java]: we're using motor speed now, still assuming a lot

**Review by bengold22 — APPROVED** [review]

File: src/main/java/frc/robot/commands/ResetRamp.java [hunk 4]
```diff
@@ -22,23 +29,28 @@ public ResetRamp(Ramp ramp) {
   @Override
   public void initialize() {
     startTime = Timer.getFPGATimestamp();
+    ramp.setMotor(0.3); //assuming positive is forward with a random speed
```
Comment (noahheller) [inline src/main/java/frc/robot/commands/ResetRamp.java]: make this a constants

Comment (neo-Aditya) [reply src/main/java/frc/robot/commands/ResetRamp.java]: Although I don't see a use for this when we don't have it refined yet, I added it

**Pull request merged**